packages/backend/src/health.test.ts

```diff
@@ -52,19 +78,47 @@ describe('health', () => {
         expect(await getHealthState(false)).toEqual({
             healthy: true,
             version: '0.1.0',
+            rudder: undefined,
             mode: LightdashMode.DEFAULT,
             isAuthenticated: false,
             needsSetup: true,
+            needsProject: false,
+            defaultProject: undefined,
+            latest: { version: Image.name },
+        });
+    });
+    it('Should return needsProject true and defaultProject if there are no projects in DB', async () => {
+        (projectService.hasProject as jest.Mock).mockImplementation(
+            async () => false,
+        );
+
+        expect(await getHealthState(false)).toEqual({
+            healthy: true,
+            version: '0.1.0',
+            rudder: undefined,
+            mode: LightdashMode.DEFAULT,
+            isAuthenticated: false,
+            needsSetup: false,
+            needsProject: true,
+            defaultProject: {
+                name: 'default',
+                type: 'dbt',
+                profiles_dir: '/',
+                project_dir: '/',
+            },
```

Comment:
I couldn't find where this mock is setup. 

just to check: this is the default project for this mock only? Usually it pulls the project from `lightdash.yaml`?